Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The line below points to distroless/base as of 2019-11-15. The SHA should be
+# kept in sycn with distroless_base definition in the WORKSPACE file.
```

Comment:
These lines seem outdated and can be removed.

Reply:
Sigh.. I think I just blindly copied the header from another k8s Dockerfile, thx!